t/020-fifottl.t

```diff
@@ -245,6 +245,22 @@ test:test('ttl after delay test', function(test)
     test:is(task.ttr, TTR * 1000000, 'check TTR after release')
 end)
 
+test:test('buried task in dropped queue', function(test)
+    test:plan(1)
+    queue.create_tube('task_line', 'fifottl', {ttr = 1, if_not_exist = true})
```

Comment:
I would use 0.1 and 0.2 seconds. I think that is enough.

Reply:
I agree, will fix